src/components/CheckBoxTree/index.tsx

```diff
@@ -99,6 +99,18 @@ class CheckBoxTree extends React.Component<CheckBoxTreeProps> {
         }
     };
 
+    getAgentTags = (agentName: string): string[] => {
+        const { treeData } = this.props;
+        const tags: string[] = [""];
+        const agent = treeData.find((agent) => agent.title === agentName);
+        if (agent) {
+            agent.children.forEach((child) => {
+                tags.push(child.value as string);
+            });
```

Comment:
Nit: Could also do a map operation here to save you a few lines!
```suggestion
            return agent.children.map((child) => child.value as string);
```

Reply:
We also need the "" in the array, so it ends up still needing a `concat` or a `push`